### PR TITLE
removes some guns and there ammo from cargo goodie

### PR DESCRIPTION
**Pull request by MissFox0810**
## About The Pull Request

Removes some guns from the goodies section of cargo, as it allows buying lethal weapons without alert restrictions.

## Why It's Good For The Game

Stops Security from buying leathals with out restriction. 

## Changelog
:cl: Miss Fox
del: Removes guns from Goodies in cargo
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/modules/cargo/code/bloat.dm

consider commenting out the stuff on the bottom there too

**Comment by Draganfrukts** [comment]
IMO This should be regulated IC, removing most of this seems like it's almost for no reason. Stuff like the mafia crate are for traitors, security can still get lethals without the need of cargo, there's no need to remove the tech shell one because they literally can just ask science to research it for a menial amount of points, mini energy gun has like 3 lethal shots before it runs out, and so on. It seems redundant at best

File: code/modules/cargo/goodies.dm [hunk 1]
```diff
@@ -76,6 +77,8 @@
 /datum/supply_pack/goody/wt550ammoap //SKYRAT EDIT ADDITION
```
Comment (Azarak) [inline code/modules/cargo/goodies.dm]:
```suggestion
```

File: code/modules/cargo/goodies.dm [hunk 2]
```diff
@@ -76,6 +77,8 @@
...
 
 /datum/supply_pack/goody/wt550ammoic //SKYRAT EDIT ADDITION
```
Comment (Azarak) [inline code/modules/cargo/goodies.dm]:
```suggestion
```

**Comment by Azarak** [comment]
> 
> 
> IMO This should be regulated IC, removing most of this seems like it's almost for no reason. Stuff like the mafia crate are for traitors, security can still get lethals without the need of cargo, there's no need to remove the tech shell one because they literally can just ask science to research it for a menial amount of points, mini energy gun has like 3 lethal shots before it runs out, and so on. It seems redundant at best

I wish it would be. Now you've got security/captain stocking up on extra guns just cause because heho private buy, or randos getting guns and then looking for trouble just cause they have a gun, like a good bit of spacetiders used to do.
@hugmeorelse what's your input, Xyel?

**Comment by Owai-Seek** [comment]
Why is mini energy gun being removed. It's literally like, the weakest gun you can get as a goodie pack. It only has like, six shots in it. You can only kill like, a single spider with it.

**Comment by Zenitheevee** [comment]
Being honest, I dont see a reason to remove these. 

My reasonings: 
Mini E-gun is Shit. Calling this "Lethal" Would be on the same standing point of calling Welders lethal.  
Secret Keeper crate... Is an Emag only crate, 100k credits **because you're suppose to dump 22 TC into just moneybags to buy it. This isn't even a fucking goodie pack THE FUCK**
And as someone who put the work into most of the things getting removed... 
I honestly feel extremely cheated if this gets passed through.

**Comment by Zenitheevee** [comment]
> 
> 
> > IMO This should be regulated IC, removing most of this seems like it's almost for no reason. Stuff like the mafia crate are for traitors, security can still get lethals without the need of cargo, there's no need to remove the tech shell one because they literally can just ask science to research it for a menial amount of points, mini energy gun has like 3 lethal shots before it runs out, and so on. It seems redundant at best
> 
> I wish it would be. Now you've got security/captain stocking up on extra guns just cause because heho private buy, or randos getting guns and then looking for trouble just cause they have a gun, like a good bit of spacetiders used to do.
> @hugmeorelse what's your input, Xyel?

Also, Literally whats stopping them from just private buying the gun crates that are, Objectionably, Cheaper and more output? 
WT550 is 2k for 1, Or 1.55k if you private order the cargo crate
Combat shotgun is 1.5, Half the price of the crate but 1/3 of the output
The laser guns are the only ones that are unique but a laser crate was, what, 800? 
Mini Egun.. again, Literal shit gun.

**Comment by Azarak** [comment]
E-gun's not bad at all if you get 2. Besides this is not a matter of whether a gun is good or not, or whether its price efficient or not, it's whether people should be able to buy guns 4 no reason, because this is what it essentially is. It's a principle of whether people(civilians) should be able to get/own a gun, which I support, but it doesn't work right now

**Comment by Azarak** [comment]
Also if you designed the crate specificlaly for someone to dump 22 TC just into money to get it, that's pretty bad design. On top of it's just a reference to old mafias.

**Comment by Owai-Seek** [comment]
Can you at the very least convert the min-egun to an armory access restricted weapon, so it can still be obtained if the warden or HoS gives someone permission to purchase one. it's one of the few low power weapons in the game that can be considered a self-defense weapon, and not a kill the valids weapons.

**Comment by Zenitheevee** [comment]
yeah... Can we atleast compromise on a mini-egun crate? 1500 for three or something like that?

**Comment by YakumoChen** [comment]
Honestly the mafia crate should just be a syndie bundle, not an emag crate.

**Review by KathrinBailey — APPROVED** [review]

**Pull request merged**